tests/unittests/Foundation/NSTimerTests.mm

```diff
@@ -105,16 +105,15 @@ - (void)dealloc {
     NSTimerTestObj* testObj = [[[NSTimerTestObj alloc] initWithValue:NO] autorelease];
     volatile long waitCompletion = 0;
 
-    auto selectorCalledAsync = std::async(std::launch::async,
-                                          [&waitCompletion, testObj]() {
-                                              BOOL fooChanged = [testObj waitOnCalledConditionForInterval:15] && [testObj called];
-                                              _InterlockedExchange(&waitCompletion, 1L);
-                                              return fooChanged;
-                                          });
+    auto selectorCalledAsync = std::async(std::launch::async, [&waitCompletion, testObj]() {
+        BOOL fooChanged = [testObj waitOnCalledConditionForInterval:2] && [testObj called];
```

Comment:
>2 [](start = 68, length = 1)

in terms of unit tests 2 seconds is still a long time. Perhaps consider something in the ms range.